assets/src/components/conversations/ConversationsPreviewList.tsx

```diff
@@ -31,6 +32,22 @@ const ConversationsPreviewList = ({
     isCustomerOnline,
   } = useConversations();
 
+  React.useEffect(() => {
+    if (
+      selectedConversationId &&
+      conversationIds.indexOf(selectedConversationId) !== -1
+    ) {
+      // Scrolls to highlighted ConversationItem component if not visible
+      const el = document.getElementById(
+        `ConversationItem--${selectedConversationId}`
+      );
+
+      if (!isScrolledIntoView(el)) {
+        el?.scrollIntoView(false);
+      }
+    }
+  }, [selectedConversationId]);
```

Comment:
note to self: eslint is throwing a warning for adding `conversationIds` as a dependency here, but we don't actually want this to rerun whenever `conversationIds` change... need to figure out the "correct" way to handle this, but otherwise we can just disable the eslint warning 🤷

Reply:
you could define `const conversationIdsIncludesSelected = conversationIds.includes(selectedConversationId)` and 

```
React.useEffect(() => {
  if (conversationIdsIncludesSelected) {}
}, conversationIdsIncludesSelected)
```